README.md

```diff
@@ -0,0 +1,223 @@
+# Sushi Mail Template - A Module for MediaKiwi to create easy to use mail templates with placeholders
+[![NuGet version (Sushi.MailTemplate)](https://img.shields.io/nuget/v/Sushi.MailTemplate.svg?style=flat-square)](https://www.nuget.org/packages/Sushi.MailTemplate/)
+[![Build status](https://dev.azure.com/supershift/Mediakiwi/_apis/build/status/Sushi.MailTemplate)](https://dev.azure.com/supershift/Mediakiwi/_build/latest?definitionId=106)
+## Features
+Sushi Mail Template is a NuGet library that allows you to easily create mail templates with placeholders, repeating placeholder groups and sections that can be .
+## Using Sushi Mail Template - Setup
+### Installing via NuGet
+The easiest way to install Sushi.MailTemplate is via [NuGet package](https://www.nuget.org/packages/Sushi.MailTemplate) to your MediaKiwi-powered solution.
```

Comment:
**MediaKiwi** is spelled with a lower k. https://github.com/Supershift/Sushi.Mediakiwi. You might wanna use the same spelling.